src/cross_section.F90

```diff
@@ -248,6 +267,12 @@ subroutine calculate_nuclide_xs(i_nuclide, i_sab, E, i_mat, i_nuc_mat, u)
       micro_xs(i_nuclide) % kappa_fission = &
            nuc % reactions(nuc % index_fission(1)) % Q_value * &
            micro_xs(i_nuclide) % fission
+
+      ! Calculate microscopic nuclide delayed nu-fission cross section
+      micro_xs(i_nuclide) % delayed_nu_fission = (ONE - f) * &
+           nuc % delayed_nu_fission(i_grid) + f * &
+           nuc % delayed_nu_fission(i_grid+1)
```

Comment:
This is also related to the performance drop. I haven't profiled anything, so I don't actually know which of these is more expensive. It may be that calculating these microscopic cross sections is more costly. Keep in mind the cost really has nothing to do with the meager amount of FLOPs needed but rather the fact that going to fetch data from `nuc%delayed_nu_fission` likely means missing cache and having to go out to main memory.